x-pack/plugins/fleet/.storybook/context/index.tsx

```diff
@@ -111,7 +110,7 @@ export const StorybookContext: React.FC<{ storyContext?: Parameters<DecoratorFn>
           writeIntegrationPolicies: true,
         },
       },
-      guidedOnboarding: guidedOnboardingMock.createStart(),
+      guidedOnboarding: {},
```

Comment:
why was this removed?

Reply:
the removed mock contains `jest` functions which break storybook. I guess the mock was intended to be used in tests, and was accidentally added to the storybook setup.